<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>luksaddkey</class>
 <widget class="QDialog" name="luksaddkey">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>342</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Add A Key To A Volume</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="labelLuksVolume">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Volume Path</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOpenFile">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>open file</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOpenPartition">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>open partition</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAdd">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>310</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Add</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>310</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="textEditPathToVolume">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Key Already In The Encrypted Volume</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string> Key To Be Added To The Encrypted Volume</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditReEnterPassphrase">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelReEnterPassphrase">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Reenter Key</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOpenNewKeyFile">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>250</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>open keyfile</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="textEditPassphraseToAdd">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelNewPassphrase">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Key</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cbNewKey">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Key</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>KeyFile</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Key+KeyFile</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="textEditExistingPassphrase">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelExistingPassphrase">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Key</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOpenExistingKeyFile">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>100</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>open keyfile</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cbExistingKey">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Key</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>KeyFile</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Key+KeyFile</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="cbVolumeType">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>321</width>
     <height>29</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>LUKS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TrueCrypt</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>VeraCrypt</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Volume Type</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPIM">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>321</width>
     <height>29</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>PIM</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>581</width>
     <height>341</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>textEditPathToVolume</tabstop>
  <tabstop>pushButtonOpenFile</tabstop>
  <tabstop>pushButtonOpenPartition</tabstop>
  <tabstop>cbExistingKey</tabstop>
  <tabstop>textEditExistingPassphrase</tabstop>
  <tabstop>pushButtonOpenExistingKeyFile</tabstop>
  <tabstop>cbVolumeType</tabstop>
  <tabstop>lineEditPIM</tabstop>
  <tabstop>cbNewKey</tabstop>
  <tabstop>textEditPassphraseToAdd</tabstop>
  <tabstop>pushButtonOpenNewKeyFile</tabstop>
  <tabstop>lineEditReEnterPassphrase</tabstop>
  <tabstop>pushButtonAdd</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
